rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -1,25 +1,24 @@
 import React, {useEffect, useState} from "react";
 import Xarrow from "react-xarrows";
-import '../stylesheets/DivGraph.css'
+import '../stylesheets/LandingPageGraph.css'
 
 //Loading Components
-import Data_Association_PC from './Data_Association_PC';
+import Bundle_Adj_PC from './Bundle_Adj_PC';
 import DivNode from './DivNode';
 import EdgeList from './gtsfm_edge_list.js';
 // import FrontendSummary from './FrontendSummary';         - add this in part 3
 // import MVOSummary from './MVOSummary';                   - add this in part 3
 //import RelativeRsDivNode from './relativeRsDivNode';      - add this in part 3
 //import RotSuccessSummary from './RotSuccessSummary';      - add this in part 3
 import SfMDataDivNode from './SfMDataDivNode';
- 
 
 //loading json files in result_metrics folders to display summary metrics
-import data_association_json from '.././result_metrics/data_association_metrics.json';
-import frontend_summary_json from '.././result_metrics/frontend_summary.json';
-import multiview_optimizer_json from '.././result_metrics/multiview_optimizer_metrics.json';
+import data_association_json from '../result_metrics/data_association_metrics.json';
```

Comment:
is there a way to just have the user run a `setup.sh` script that generates a symbolic link, so that only one copy of the files is needed?

```bash
ln -s result_matrics rtf_vis_tool/result_metrics
```
or something like that?

Reply:
After looking more into Linux symbolic links, this looks like a great way to get around saving two copies of the file. I'll add a `setup.sh` file within the `rtf_vis_tool` directory that creates symbolic links to the `results` and `result_metrics` directories.

Reply:
I've gotten a Linux symlink to connect the directories successfully, but React is still having issues with it. Messaged you privately about it in more detial.